### PR TITLE
Add GitHub workflow for publishing libClusterFuzz.

### PR DESCRIPTION
**Pull request by oliverchang**
Triggered whenever we create a release.

### CODE REVIEW

File: .github/workflows/publish-to-pypi.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    - name: Install pypa/build
+      run: >-
+        python -m
+        pip install
+        build
+        --user
+    - name: Build a binary wheel and a source tarball
+      run: ./build.sh
+    - name: Publish distribution to PyPI
+      uses: pypa/gh-action-pypi-publish@master
```
Comment (inferno-chromium) [inline .github/workflows/publish-to-pypi.yaml:44]: Best to pin these by hashes, same above as well (instead of version or master).

**Review by inferno-chromium — APPROVED** [review]

File: .github/workflows/publish-to-pypi.yaml (hunk 1 above)
Comment (oliverchang) [reply .github/workflows/publish-to-pypi.yaml]: done!

**Comment by jonathanmetzman** [comment]
LGTM. Thanks!

**Pull request merged**